folio-kafka-wrapper/src/main/java/org/folio/kafka/cache/KafkaInternalCache.java

```diff
@@ -67,6 +118,7 @@ public boolean containsByKey(String key) {
   /**
    * Clean Kafka cache from outdated elements.
    * Outdated element - element difference between its value and current time is more or equal to the eventTimeoutHours.
+   *
    * @param eventTimeoutHours - timeout for event in hours.
    */
   public void cleanupEvents(int eventTimeoutHours) {
```

Comment:
need to rewrite this method

Reply:
added verification that the event.value is not null